src/Components/MapContainer/MapContainer.js

```diff
@@ -20,30 +13,32 @@ function MapContainer({restaurants}) {
     res += (restaurant.lat / restaurants.length)
     return res
   }, 0)
-  const [geojson, setGeojson] = useState({});
+  const [pins, setPins] = useState(null)
+  const [popupInfo, setPopupInfo] = useState(null);
   const viewport = {
     longitude: aveLon,
     latitude: aveLat,
     zoom: 11
   };
 
-  useEffect(() => {
-    let pinArray = []
-    restaurants.map(restaurant => {
-      let pinObject = {type: 'Feature', geometry: {type: 'Point', coordinates: [restaurant.lon, restaurant.lat]}}
-      return pinArray.push(pinObject)     
-    });
-
-    let objectCopy = {
-      type: 'FeatureCollection',
-      features:[
-        {type: 'Feature', geometry: {type: 'Point', coordinates: [-122.4, 37.8]}}
-      ] 
-    }
-    objectCopy.features = pinArray
-    setGeojson({...objectCopy})
-  },[restaurants])
-
+ useEffect(() => {
+      const res = restaurants.map((restaurant) => {
+        return <Marker
```

Comment:
This is so neat. It seems like Mapbox has several alternative ways of formatting pin data?